cosmos/airflow/graph.py

```diff
@@ -11,6 +11,7 @@
 
 from cosmos.config import RenderConfig
 from cosmos.constants import (
+    DBT_COMPILE_TASK_ID,
```

Comment:
This constant currently has the value `dbt_compile`. It would be great if we could stick to a name both from a user-interface and an internal implementation perspective. We are naming this `setup_task` in several parts of the code. Do we prefer `async_setup` or `dbt_compile`? Now is a good time to pick up one and stick to them

Reply:
Some examples: `SetupAsyncOperator`, `_add_dbt_compile_task`,` enable_setup_task`, `DBT_COMPILE_TASK_ID`

Reply:
changed to SetupAsyncOperator and task-id: `dbt_setup_async`